kedro/template/{{ cookiecutter.repo_name }}/kedro_cli.py

```diff
@@ -122,9 +122,9 @@ def test(args):
 
 @cli.command()
 def install():
-    """Install project dependencies from requirements.txt."""
+    """Install project dependencies from both requirements.txt and requirements.yml."""
     python_call("pip", ["install", "-U", "-r", "src/requirements.txt"])
-
+    call("conda", ["install", "--file src/requirements.yml", "--yes"])
```

Comment:
What will happen if the user manages their python environments with something else than `anaconda`? Could you think of a way where we can still have this functionality and not enforce users to use `anaconda` and `requirements.yml` files?

Reply:
@idanov - not a bad idea. At the moment since requirements.yml is being included in the project template then this might be an issue because it'll be there by default so it'll be there regardless of someone using anaconda or not. 

But we could change it to be an opt-in process for installing using requirements.yml. So if someone goes through the step to create their own requirements.yml then the conda install would run. **Is that what you're meaning?**

Reply:
There's also the matter of existing projects that are upgrading, we wouldn't want it to suddenly start failing for them.

Reply:
@evanmiller29 yep, I think it might be best if we make it an opt-in feature as you describe it. This way we don't break it for anyone, not enforce `conda` on users and if they want to use it for `conda`, they can easily do so. However, we should document it somewhere - maybe in the help for `kedro install`?

Reply:
@idanov - yep, just going to push the code and update the documentation now. 

For the tests - if they are needed I might need a little help writing them as I've never done any of this magnitude. Is there any resources you might recommend?